extension/src/lib/chromeSettings.js

```diff
@@ -0,0 +1,80 @@
+/*globals fluid, gpii */
+
+"use strict";
+
+fluid.defaults("gpii.chrome.settings", {
+    gradeNames: "fluid.modelComponent",
+    components: {
+        chromeVox: {
+            type: "gpii.chrome.extensionHolder",
+            options: {
+                extensionId: "kgejglhpjiefppelpmljglcjbhoiplfn",
```

Comment:
Yes,

it's ChromeVox' id, see:  https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en